src/themes/arctic/theme.css

```diff
@@ -0,0 +1,1264 @@
+    border: none;
+    padding-left: 1.5rem !important;
+    padding-right: 1.5rem !important;
+    transition: none !important;
+    border-radius: 999999px;
+}
+
+.collapseContent {
+    margin-top: 0.5rem;
+    border-radius: 2rem;
+    background-color: rgb(var(--bg-light));
+}
+
+/* #region emby-collapse-has */
+/* .emby-collapse:has(.collapseContent.hide) .emby-collapsible-button { */
+/*     border-radius: 999999px; */
+/*     transition: none !important; */
+/* } */
+
+/* .emby-collapse:has(.collapseContent.expanded) .emby-collapsible-button { */
+/*     border-top-left-radius: 2rem; */
+/*     border-top-right-radius: 2rem; */
+/*     border-bottom-left-radius: 0; */
+/*     border-bottom-right-radius: 0; */
+/* } */
+/* .emby-collapse:has(.collapseContent.expanded) .collapseContent { */
+/*     border-bottom-left-radius: 2rem; */
+/*     border-bottom-right-radius: 2rem; */
+/* } */
+/* #endregion */
+
+.formDialogHeader:not(.formDialogHeader-clear),
+.skinHeader-withBackground {
+    color: var(--text);
+    background: rgb(var(--bg-light));
+}
+
+.skinHeader {
+    width: calc(100% - 2rem);
+    margin: 0 auto;
+    border-radius: 1rem;
+    padding: 0.75rem;
+    box-sizing: border-box;
+    top: 0.5rem;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+    transition: transform 0.1s linear;
+}
+
+.skinHeader.headroom--unpinned {
+    transform: translateY(calc(-100% - 1rem));
+}
+
+.videoPlayerContainer ~ .skinHeader {
+    background: transparent !important;
+}
+
+.libraryPage:not(.noSecondaryNavPage) {
+    /* padding-top: 7.5em !important; */
+    padding-top: 10em !important;
+}
+
+.libraryPage .editPageSidebar {
+    top: 6.5rem;
+    height: calc(100% - 7rem);
+    border-color: rgb(var(--nord2));
+}
+
+.libraryPage .editPageInnerContent {
+    padding-top: 0;
+    padding-right: 1rem;
+    box-sizing: border-box;
+}
+
+.libraryPage .editPageInnerContent > .formDialogHeader {
+    border-radius: 1rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+}
+
+.libraryPage.metadataEditorPage {
+    padding-top: 6.5em !important;
+}
+
+.skinHeader.skinHeader-blurred {
+    background: rgba(var(--bg-light), 0.9);
+    backdrop-filter: blur(12px);
+}
+
+.pageTitleWithDefaultLogo {
+    background-image: url(../../assets/img/banner-light.png);
+}
+
+.layout-tv .pageTitleWithDefaultLogo {
+    background-image: url(../../assets/img/icon-transparent.png);
+}
+
+@media screen and (max-width: 380px) {
+    .pageTitleWithLogo {
+        display: none;
+    }
+
+    .pageTitle {
+        display: none;
+    }
+}
+
+@media screen and (min-width: 100em) {
+    .skinHeader .headerTop {
+        padding: 0 !important;
+    }
+
+    .skinHeader .headerTabs {
+        margin-top: -2.75rem;
+    }
+
+    .skinHeader .headerTabs .emby-tabs {
+        padding: 0 !important;
+    }
+}
+
+html {
+    color-scheme: dark;
+    background: rgb(var(--bg));
+    font-family: Nunito, Noto Sans, Noto Sans HK, Noto Sans JP, Noto Sans KR,
```

Comment:
Very nice theme! However, I found some problems while using it. Some same characters have different glyphs in "CJK", such as ["台" or "骨"](https://fonts.google.com/?preview.text=%E5%8F%B0%E9%AA%A8&preview.text_type=custom&query=Noto+Sans&category=Sans+Serif), so if we specify the fallback order of Noto Sans family here, it may make some characters look weird.

I'm not good at CSS, but I'm wondering could this problem be solved.

Reply:
Ah, yeah this is my bad as I didn't test with non-english fonts. I had thought about it but was going to leave it for later depending on whether this PR would proceed or not. I added Nunito here purely because I love the font, but it can totally be removed here to return to Noto Sans.

I'm not sure if the CJK variants include other glyphs that would overlap with latin characters, but if they don't then you may be right that the order could be swapped around.